PendingReleaseNotes.md

```diff
@@ -0,0 +1,5 @@
+>=20.11.2
+
+--------
+
+* fixed duplicated partitions & topics in fetch response
```

Comment:
can you add a link to the pr and the issue? these are going to make building release notes much easier.